deploy/Dockerfile_debug

```diff
@@ -27,6 +27,8 @@ RUN go get -u github.com/awslabs/amazon-ecr-credential-helper/ecr-login/cli/dock
 RUN make -C /go/src/github.com/awslabs/amazon-ecr-credential-helper linux-amd64 
 COPY . .
 RUN make GOARCH=${GOARCH} && make out/warmer
+# Blank out the credentials file
+RUN echo "{}" > /root/.docker/config.json
```

Comment:
I believe a better approach would be to remove the line RUN docker-credential-gcr configure-docker which is the line that adds .docker/config.json